content/en/docs/Getting Started/tasks.md

```diff
@@ -0,0 +1,168 @@
+---
+title: "Getting started with Tasks"
+likTitle: "Tasks"
+weight: 1
+description: >
+  Set up and run your first Tekton task
+---
+
+This tutorial shows you how to 
+
+1. Create a Kubernetes cluster with [minikube](https://minikube.sigs.k8s.io/).
+1. Install Tekton pipelines.
+1. Create a task.
+1. Use `TaskRun` to instantiate and run your task.
+
+## Prerequisites
+
+1.  [Install minikube](https://minikube.sigs.k8s.io/docs/start/). You only have
+    to complete the step 1, "Installation".
+1.  [Install kubectl](https://minikube.sigs.k8s.io/docs/start/)
+
+## Create your Kubernetes cluster
+
+Create a cluster
+
+```bash
+minikube start
+```
+
+The process takes a few seconds, you see an output symilar to the following,
+depending on the [minikube driver](https://minikube.sigs.k8s.io/docs/drivers/)
+that you are using:
+
+<pre>
+😄  minikube v1.25.1
+✨  Using the docker driver based on existing profile
+👍  Starting control plane node minikube in cluster minikube
+🚜  Pulling base image ...
+🔄  Restarting existing docker container for "minikube" ...
+🐳  Preparing Kubernetes v1.23.1 on Docker 20.10.12 ...
+    ▪ kubelet.housekeeping-interval=5m
+    ▪ Generating certificates and keys ...
+    ▪ Booting up control plane ...
+    ▪ Configuring RBAC rules ...
+🔎  Verifying Kubernetes components...
+    ▪ Using image gcr.io/k8s-minikube/storage-provisioner:v5
+🌟  Enabled addons: storage-provisioner, default-storageclass
+🏄  Done! kubectl is now configured to use "minikube" cluster and "default" namespace by default
+</pre>
+
+You can check that the cluster was successfully created with `kubectl`:
+
+```bash
+kubectl cluster-info
+```
+
+The output confirms that Kubernetes is running:
+
+<pre>
+Kubernetes control plane is running at https://127.0.0.1:39509
+CoreDNS is running at
+https://127.0.0.1:39509/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+
+To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+</pre>
+
+## Install Tekton Pipelines
+
+1. To install the latest version of Tekton Pipelines, use `kubectl`:
+
+   ```bash
+   kubectl apply --filename \
+   https://storage.googleapis.com/tekton-releases/pipeline/latest/release.yaml
+   ```
+
+1. Monitor the installation:
+
+   ```bash
+   kubectl get pods --namespace tekton-pipelines --watch
+   ```
+
+When all components show `Running` under the `STATUS` column the installation
+is complete.
+
+Hit *Ctrl + C* to stop monitoring.
+
+## Create and run a basic task
+
+A **task**, represented in the API as an object of kind `Task`, defines a
+series of **steps** that run sequentially to perform some build work. Every
```

Comment:
nit: maybe instead of "build work" something more generic (but CI/CD specific), something like "perform logic the Task requires" or drop that part of the sentence entirely? (cuz not every Task is doing something build related)